### PR TITLE
update grafana-cloud-metrics.yaml to install and configure grafana 3.…

### PR DESCRIPTION
**Pull request by VinnyQ**
# What

Update grafana-cloud-metrics.yaml to install and configure grafana 3.1.1 to integrate with blueflood rackspace metrics.

# Why

Because, progress.

# How

Removing instructions for previous version of grafana and add instruction to install grafana 3.1.1 from binary package, add blueflood plugin as a datasource, install graphite-api, install blueflood-graphite-api-finder, all proxied by nginx, and use grafana api to setup the datasource and admin user.

Also moved the previous (grafana 1.8.1) version, in its entirety, to a new file:  grafana-cloud-metrics-1.8.1.yaml

# Test

Create a custom heat template in Reach - Orchestration (mycloud.rackspace.com) using the content of the yaml, spin up a stack using it with the correct auth info, and verify grafana3 is installed and working.

### CODE REVIEW

**Comment by VinnyQ** [comment]
@wolfdancer shane, here's the new version of the grafana yaml that will install and configure grafana 3.1.1 to work with blueflood.  Give it a try!

@goru97 please review also if you have a chance.  Thanks!

**Comment by VinnyQ** [comment]
@wolfdancer do note that I moved the previous version (grafana 1.8.1) to a totally new file:  grafana-cloud-metrics-1.8.1.yaml.

Once we merge, any links to the grafana-cloud-metrics.yaml will see and use the grafana 3.1.1 install instead.  To test from this PR, click on the "View" button in the diff view for the grafana-cloud-metrics.yaml file.

**Comment by VinnyQ** [comment]
Oh great news about this version, we get to completely remove the dependencies on the blueflood git repo (https://github.com/rackerlabs/blueflood.git) since we no longer need the config.js file from it (all grafana plugin config are done via the blueflood datasource and the grafana http api instead).

**Comment by wolfdancer** [comment]
@VinnyQ nice!  I am checking it out now.

**Review by wolfdancer — CHANGES_REQUESTED** [review]

File: grafana-cloud-metrics.yaml
Comment (VinnyQ) [inline grafana-cloud-metrics.yaml]: no that's the heat template version we're using.  Apparently they versioned template syntax by date.

Comment (wolfdancer) [inline grafana-cloud-metrics.yaml]: Ah.  Thanks.

**Comment by VinnyQ** [comment]
@wolfdancer are you good with this PR?  If so go ahead and +1 it and I'll merge it to master.

**Comment by wolfdancer** [comment]
+1 from Product Perspective.  I don't know enough to review the template code, though. :)

**Comment by VinnyQ** [comment]
Np, most of the script is actually bash and shell commands, and as long as the end result matches and behave as you expected, then we're good here :)  Thanks for the +1, I am merging now.

**Comment by VinnyQ** [comment]
@wolfdancer actually I don't have write access to this repo.  Either you or @goru97 or @chrishultin or @rs-randallburt will have to merge this for us.  Can one of you merge for me?  thanks!

**Comment by wolfdancer** [comment]
I don't have permission to merge either :)

**Comment by chrishultin** [comment]
I can go ahead and merge this in the morning.  Can either of you remind me if this needs to be pushed to the public catalog or not?

On Nov 2, 2016, at 5:27 PM, Shane Duan <notifications@github.com<mailto:notifications@github.com>> wrote:


I don't have permission to merge either :)

**Comment by wolfdancer** [comment]
Thanks @chrishultin.  

By pushing to the public catalog you meant the heat templates available for all cloud accounts out of the box, right?  If so this has not been pushed there (we would love it though). :)

At the moment, it is still a custom template that user needs to create.

**Comment by chrishultin** [comment]
I've gone ahead and merged.

**Comment by wolfdancer** [comment]
Thanks!

**Pull request merged**
